docs/administrator/identity/service.md

```diff
@@ -5,8 +5,27 @@ Title: Identity Service
 # Identity Service
 The [Identity Service](https://docs.alfresco.com/identity/concepts/identity-overview.html) is used by Activiti Enterprise for authorization and to manage user and role management.
 
-You can log into the administration console for the Identity Service by navigating to `identity.{domain-name}/auth` and using the username **admin** and password of **admin**.
+You can log into the administration console for the Identity Service by navigating to `identity.{domain-name}/auth/admin` and using the username **admin** and password of **admin**.
 
 **Note**: it is strongly recommended that you update the password for the administration console when you first log into it. 
 
-A default realm of **alfresco** is already setup with the installation of Alfresco Activiti Enterprise. 
\ No newline at end of file
+A default realm of **alfresco** is already setup with the installation of Alfresco Activiti Enterprise. 
+
+## Improving API call performance 
+It is possible to improve the performance of API calls to the runtime bundle, query service and audit service by including a user's group membership in the JSON Web Token (JWT). 
+
+**Note**: Adding group membership to the JWT increases its size. It is advisable to add group membership only when there are not an excessive number of groups per user, otherwise the performance will be degraded rather than improved.
```

Comment:
@m-hulbert, @mteodori, we should add a note about the possible security risk as well.